etc/concourse/scripts/cf-test-test

```diff
@@ -1,14 +1,80 @@
 #!/bin/bash
 
-set -e
+set -e -x
+
+APPS=()
```

Comment:
We should extract `getApps` and `restartApps` to enable their reuse across scripts